safe-defaults/tmpfiles-enable.conf.in

```diff
@@ -6,7 +6,9 @@
 # if a symlinked config file in the ostree is no longer needed it should
 # be made empty rather than deleted, otherwise the change from L to r
 # in this file will be ineffective.
+L @SYSCONFDIR@/chromium-browser/policies/managed/01_doh.json - - - - @DATAROOTDIR@/eos-safe-defaults/chrome/policies/managed/01_doh.json
 L @SYSCONFDIR@/NetworkManager/conf.d/opendns-family-shield.conf - - - - @DATAROOTDIR@/eos-safe-defaults/nm-opendns-family-shield.conf
+L @SYSCONFDIR@/opt/chrome/policies/managed/01_doh.json - - - - @DATAROOTDIR@/eos-safe-defaults/chrome/policies/managed/01_doh.json
 L @SYSCONFDIR@/opt/chrome/policies/managed/safe-search.json - - - - @DATAROOTDIR@/eos-safe-defaults/chrome/policies/managed/chrome-safe-search.json
 D /run/flatpak/extension/org.chromium.Chromium.Policy.safe-search 0755 - - -
```

Comment:
This extension point is now misleadingly named, but I think we should leave it be for now.